tools/llvm-spirv/llvm-spirv.cpp

```diff
@@ -587,6 +593,17 @@ int main(int Ac, char **Av) {
     }
   }
 
+  if (SPIRVReplaceLLVMFmulAddWithOpenCLMad.getNumOccurrences() != 0) {
+    if (IsReverse) {
+      errs() << "Note: --spirv-replace-fmuladd-with-ocl-mad option ignored as "
+                "it only "
+                "affects translation from LLVM IR to SPIR-V";
```

Comment:
```suggestion
                "it only affects translation from LLVM IR to SPIR-V";
```

Reply:
Applied in 05d38e03deb11d274068805048ddbab164455924